app/api/logging/__init__.py

```diff
@@ -82,13 +69,4 @@ def init(program_name: str) -> None:
             ),
         },
     )
-    logger.info("invoked as: %s", " ".join(original_argv))
-
-
-def get_logger(name: str) -> logging.Logger:
-    """Return a logger with the specified name."""
-    return logging.getLogger(name)
```

Comment:
For context, the original implementation had an additional check to try to encourage correct usage. It was like this:

```python
    if not name.startswith("massgov."):
        raise ValueError(
            "invalid logger name %r (try passing __package__ instead of __name__)" % name
        )
```

Reply:
thanks for the context! with the handler now attached to the root logger, that's no longer necessary i assume?